src/UI/ValueFactory.cs

```diff
@@ -388,5 +406,33 @@ private static bool GetNewColorSchemeValue(Design design, out object? newValue)
                 return false;
             }
         }
+
+        /// <summary>
+        /// Returns true if path is to a directory.
+        /// Thanks: https://stackoverflow.com/a/19596821/4824531
+        /// </summary>
+        /// <param name="path"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        static bool IsPathDirectory(string path)
+        {
+            if (path == null) throw new ArgumentNullException("path");
+            path = path.Trim();
+
+            if (Directory.Exists(path))
+                return true;
+
+            if (File.Exists(path))
```

Comment:
This is redundant, considering other comments here.

If the above was false, we don't care if it's a file - it's still not a directory.